### PR TITLE
LPS-163222 Add expection when form payload exceededs maximum allowed

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://issues.liferay.com/browse/LPS-163222

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf [hunk 1]
```diff
@@ -31,6 +31,10 @@
 
 		<liferay-ui:error exception="<%= DDMFormLayoutValidationException.class %>" message="please-enter-a-valid-form-layout" />
 
+		<liferay-ui:error exception="<%= LiferayFileItemException.class %>">
+			<liferay-ui:message arguments="<%= LanguageUtil.formatStorageSize(FileItem.THRESHOLD_SIZE, locale) %>" key="please-enter-valid-content-with-valid-content-size-no-larger-than-x" translateArguments="<%= false %>" />
+		</liferay-ui:error>
+
 		<liferay-ui:error exception="<%= DDMFormLayoutValidationException.MustNotDuplicateFieldName.class %>">
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf:38]: @murilostodolni great job.. I would just change the order of the exceptions to be in alphabetical order, i.e. `LiferayFileItemException` after `FormInstanceSettingsStorageTypeException` and before the `StorageException`

Comment (murilostodolni) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf:38]: Perfect!

**Review by carolmariaabb — APPROVED** [review]

**Comment by murilostodolni** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward
ci:forward

**Comment by murilostodolni** [comment]
ci:forward

**Pull request closed without merging**